app/helpers/application_helper.rb

```diff
@@ -1 +1,10 @@
-module ApplicationHelper; end
+module ApplicationHelper
+  def full_title page_title = ""
+    base_title = t("titlebooksampleapp")
```

Comment:
```suggestion
    base_title = t "titlebooksampleapp"
```
nếu key có trên 2 từ thì viết theo format `title_book_sample_app`